src/libraries/System.Collections.Immutable/tests/ImmutableArrayTest.cs

```diff
@@ -1260,15 +1426,47 @@ public void RemoveDefaultInvalid()
         }
 
         [Theory]
-        [MemberData(nameof(RemoveRangeIndexLengthData))]
+        [MemberData(nameof(RangeIndexLengthData))]
         public void RemoveRangeIndexLength(IEnumerable<int> source, int index, int length)
         {
             var array = source.ToImmutableArray();
             var expected = source.Take(index).Concat(source.Skip(index + length));
             Assert.Equal(expected, array.RemoveRange(index, length));
         }
 
-        public static IEnumerable<object[]> RemoveRangeIndexLengthData()
+        [Theory]
+        [MemberData(nameof(RangeIndexLengthData))]
+        public void AsSpanStartLength(IEnumerable<int> source, int start, int length)
```

Comment:
These two `AsSpan` tests are sandwiched between the `RemoveRange` test methods. I would recommend collocating them with the other `AsSpan` tests higher up in the test class.

Reply:
Moved.